src/core/sagas/utils.js

```diff
@@ -1,9 +1,7 @@
-/* @flow */
```

Comment:
Heh, I like the idea but an ES6 newcomer is likely to be intimidated by flow's syntax.

Now we're returning all of the state object in the new `getState` function. I guess I can add `Object` to the rest of the state objects (or omit them as you did below) but that feels even worse: we are half-documenting the keys returned by the helper.